docs/pages/access-controls/sso.mdx

```diff
@@ -249,6 +249,17 @@ or </ScopedBlock>create a `cluster_auth_preference` resource.
       # Set as saml, oidc, or github
       type: saml|oidc|github
   ```
+
+  <Admonition type="note" title="IdP-initiated SSO">
```

Comment:
- I think we should use a partial, since we include this Admonition in a number of locations

- I think a `Details` box would work better than an `Admonition` for the details about IdP-initiated SSO. It seems like this is an option that only a subset of users will use, and currently the Admonition takes up most of the space within this `TabItem`. 

- An example of using `allow_idp_initiated` in the auth connector configuration would be good since YAML is so sensitive to consistent whitespace.

Reply:
Amended, could you take another look now? @ptgott